components/tikv_util/src/lru.rs

```diff
@@ -174,14 +174,37 @@ impl<K, V> SizePolicy<K, V> for CountTracker {
     }
 }
 
-pub struct LruCache<K, V, T = CountTracker>
+pub trait GetTailKv<K, V> {
```

Comment:
There may be a comment explaining the purpose of the trait. It's unclear from this PR now.